pkg/cloudprovider/providers/gce/gce.go

```diff
@@ -779,8 +780,8 @@ func makeFirewallName(name string) string {
 	return fmt.Sprintf("k8s-fw-%s", name)
 }
 
-func makeFirewallDescription(ipAddress string) string {
-	return fmt.Sprintf("KubernetesAutoGenerated_OnlyAllowTrafficForDestinationIP_%s", ipAddress)
+func makeFirewallDescription(serviceName, ipAddress string) string {
+	return fmt.Sprintf("Kubernetes_%s_OnlyAllowTrafficForDestinationIP_%s", serviceName, ipAddress)
```

Comment:
@ArtfulCoder 

I think we actually want two tokens in the description here: the existing one is coded into a firewall manager app, and I don't know that we want to insert user-provided strings in the middle.  Maybe we should formalize the "grammar" here.

Propose changing this to a JSON label syntax.  In this case we could have `{"kubernetes.io/service-ip":"%s", "kubernetes.io/service-name":"%s"}` as the contents.

Abhi, what do you think?

@a-robinson who has touched other GCE API stuff.  Proposal is to store kube-style labels in JSON in .description.